setup.py

```diff
@@ -198,7 +198,7 @@ def get_local_version(version: "ScmVersion", time_format="%Y%m%d") -> str:
         "torch>=1.4.0",
         "tqdm",
         "scikit-learn>=0.21.2",
-        "seals==0.1.4",
+        "seals==0.1.5",
```

Comment:
Yeah, I agree. We decided in another PR to use `==` for test-only dependencies that had `0.x` revisions because semantic versioning says they can make breaking API changes at any time. However, now that it's an install requirement not a test requirement we should be permissive with versions so as not to cause conflicts for our users. With seals in particular if we make a breaking change there and forget to update `imitation` I have limited sympathy for us!

I will note that we'll still need to bump the version in `setup.py` for any changes that are _needed_ in imitation: CI doesn't install new versions automatically, only when `setup.py` is changed invalidating the existing cache.